foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/platform/database/DerbyPlatform.java

```diff
@@ -518,6 +518,8 @@ protected ExpressionOperator derbyExtractOperator() {
 
             // QUARTER emulation: ((MONTH(:first)+2)/3)
             private final String[] QUARTER_STRINGS = new String[] {"((MONTH(", ")+2)/3)"};
+            // CAST as FLOAT
+            private final String[] CAST_FLOAT = new String[] {"CAST(", " AS FLOAT)"};
```

Comment:
why not static?

Reply:
Because it causes build error in inner anonymous class. I'll refactor it to be a static class.

Reply:
Moved to static class and changed to static.